docs/content/latest/benchmark/tpcc-ysql.md

```diff
@@ -39,6 +39,10 @@ $ tar -zxvf tpcc.tar.gz
 $ cd tpcc
 ```
 
+{{< note title="Note" >}}
+The binaries are compiled with JAVA 13.
```

Comment:
Maybe be a bit more explicit? Say:
```
It is recommended to run these binaries with Java 13. Note that this package currently does not work with Java 14.
```

Also, would something compiled with Java 14 work with Java 13? If so we should just upload newer binaries and remove this note later.